components/Form.tsx

```diff
@@ -1,11 +1,64 @@
+import Image from "next/image";
+
 const Form = () => {
-    return <form className={`flex flex-col justify-between items-center h-full w-full text-white mt-10`}>
-        Contact us to book a slot
-        <input className={`mt-2 mb-2`}/>
-        <input className={`mt-2 mb-2`}/>
-        <input className={`mt-2 mb-2`}/>
-        <input className={`mt-2 mb-2`}/>
-        <button type="submit" className={`text-orange rounded bg-black pt-2 pb-2 pl-2 pr-2`}>Paw us</button>
-    </form>
-}
-export default Form;
\ No newline at end of file
+  return (
+    <div className="flex flex-col h-full w-full mt-10 pl-24 pr-24 text-black">
+      <div className="bg-form rounded-3xl p-5 flex-col flex shadow-xl">
+        <h2 className="text-3xl font-extrabold mb-4">Контактирајте нѐ!</h2>
+        <p className="mb-6">Поставете прашање или она што Ве интересира</p>
+        <div className="flex justify-between mb-6">
+          <input
+            type="text"
+            className="rounded-xl mr-3 p-5 w-1/3 border bg-form placeholder-gray"
+            placeholder="Име и презиме"
+            required
+          />
+          <input
+            type="tel"
+            className="rounded-xl mr-3 p-5 w-1/3 border bg-form placeholder-gray"
+            placeholder="Телефонски број"
+            required
+          />
+          <input
+            type="text"
+            className="rounded-xl p-5 w-1/3 border bg-form placeholder-gray"
+            placeholder="Наслов на порака"
+            required
+          />
+        </div>
+        <div className="mb-6 flex">
+          <textarea
+            className="rounded-xl p-5 w-2/3 h-70 bg-form border placeholder-gray"
+            placeholder="Вашата порака"
+            required
+          ></textarea>
+          <div className="p-3 w-1/3 h-full flex ">
+         
+            <Image
+             src="/Form images/Rectangle 22.png"
```

Comment:
Instead of referencing the picture like this and using approximate values for height/width, it's better if you import the picture (on the top of the component) and the just reference it here.